src/main/resources/solidity/Greeter.sol

```diff
@@ -0,0 +1,40 @@
+pragma solidity ^0.4.25;
```

Comment:
should this be `src/main/solidity` instead?

Reply:
Refactored